vendor/vendor.json

```diff
@@ -39,916 +39,924 @@
 			"revisionTime": "2017-07-27T15:54:43Z"
 		},
 		{
-			"checksumSHA1": "Fs3DE5rdov9GRx+jUJ7VfG/EBvA=",
+			"checksumSHA1": "pyVcj3els/KPpbHwCTcsW4jbLrE=",
+			"path": "github.com/aws/aws-sdk-go",
```

Comment:
This looks like this was accidentally vendored without the trailing slash before the `...` -- hence all the extra root SDK files. Probably simpler to just redo this with `govendor fetch github.com/aws/aws-sdk-go/...@v1.13.32`

Reply:
Solid - @bflad should be good to go with the latest commit. Thanks for looking this over!